corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -1008,6 +1029,12 @@ def column_fieldset(self):
     @memoized
     def initial_columns(self):
         if self.existing_report:
+            reverse_agg_map = {
+                'avg': 'Average',
+                'sum': 'Sum',
+                'count': 'Count',
+                'simple': 'No Aggregation'
+            }
             cols = []
```

Comment:
Yeah, we want to look at count differently in the future, so we are leaving that out.  "Count per Choice" works here, my original concern was that not leaving a "None" option or something for users might have been a bit confusing - this probably isn't an issue though, in light that expanded is the default anyway.  So, I'm good to go with this.  I have one more comment but I will make it below.